.editorconfig

```diff
@@ -0,0 +1,19 @@
+[*]
+charset=utf-8
```

Comment:
Did you mean to have this file at the top-level?
Could you document somewhere how to use these files?